src/lib/skilling/skills/hunter/birdHouseTrapping.ts

```diff
@@ -178,4 +179,182 @@ const birdhouses: Birdhouse[] = [
 	}
 ];
 
+export const birdhouseSeeds = [
+	{
+		name: 'Hammerstone seed',
```

Comment:
Im not sure if youre still working on it, but the names are still here.

Reply:
Sorry, completely missed this part! I've removed the variable and referenced `item.name` instead.